colordeducto-monitors.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/game.py b/game.py
+index 9684c6c..0b41d12 100644
+--- a/game.py
++++ b/game.py
+@@ -198,6 +198,7 @@ LEVELS_FALSE = ['def generate_pattern(self):\n\
+                 0, 10, 5, 5, 3, 3))\n\
+     return dot_list\n']
+ 
++from sugarapp.helpers import PrimaryMonitor
+ 
+ from gi.repository import Gdk, GdkPixbuf, GObject, Gtk
+ import cairo
+@@ -243,8 +244,8 @@ class Game():
+ 
+         self._canvas.connect("draw", self.__draw_cb)
+ 
+-        self._width = Gdk.Screen.width()
+-        self._height = Gdk.Screen.height() - (GRID_CELL_SIZE * 1.5)
++        self._width = PrimaryMonitor.width()
++        self._height = PrimaryMonitor.height() - (GRID_CELL_SIZE * 1.5)
```

Comment:
It probably not related to the port, but can you tweak the height ? So it doesn't look like

![Screenshot from 2020-01-05 10-54-46](https://user-images.githubusercontent.com/481418/71781209-cd16f100-2faa-11ea-8713-9f1f99754d41.png)

Reply:
I think it might have been kept on purpose, we might want to ask Walterbender, it looks like this on python2 too